packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -28,3 +28,19 @@ export function getBlockExplorerTxLink(network: Network, txnHash: string) {
 
   return blockExplorerTxURL;
 }
+
+export function getNetworkColor(chainName: string | undefined) {
```

Comment:
Please let me know what you think about it : 
used to diplay networks name with desired color : 
![Screenshot 2022-12-27 at 5 38 17 PM](https://user-images.githubusercontent.com/80153681/209664629-469d2502-220e-48ec-bd9c-626f2e16c994.jpg)

Should we add [`NETWORKS`](https://github.com/scaffold-eth/scaffold-eth/blob/923cff6aab3b52589a8f128dc42d142b7791e702/packages/react-app/src/constants.js#L26) similar to scaffold-eth v1 ?